testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -1230,6 +1230,35 @@ testModule('position', moduleConfig, () => {
 
         assert.strictEqual($content.position().top, event.target.position().top, 'overlay is positioned correctly');
     });
+
+    test('overlay content should have correct position when position.boundary changes to window', function(assert) {
+        const $overlay = $('<div>').appendTo('#qunit-fixture');
+        const $boundary = $('<div>').css({ width: 20, height: 30 }).appendTo('#qunit-fixture');
+        const widthContent = 25;
+        const heightContent = 35;
+
+        $overlay.dxOverlay({
```

Comment:
We don't use `$overlay` so we can save only link to instance
```javascript
const overlay = $('#overlay').dxOverlay({
...
}).dxOverlay('instance');
```

Reply:
done